TerrorConsole/Assets/_Main/Scripts/Player/PlayerController.cs

```diff
@@ -66,5 +66,10 @@ private void ResumeInput()
         {
             _freezeInput = false;
         }
+        public void SetMovementEnabled(bool enabled)
```

Comment:
This script already has `ResumeInput` and `StopInput`, why not use them instead?